src/genophenocorr/analysis/_commie.py

```diff
@@ -106,15 +106,22 @@ def __init__(self, cohort: Cohort,
                  missing_implies_excluded: bool = False,
                  include_sv: bool = False,
                  p_val_correction: typing.Optional[str] = None,
-                 min_perc_patients_w_hpo: typing.Union[float, int] = .1):
+                 min_perc_patients_w_hpo: typing.Union[float, int] = .1,
+                 recessive: bool = False):
```

Comment:
Whether we do a recessive, dominant, or other analysis is not a concern of `CohortAnalysis`, and, therefore, does not belong here. Checking the mode of inheritance and binning the patients accordingly is a responsibility of the predicates.  

We should discuss this during Zoom, but I think all changes related to `recessive` should be removed.